pkg/command/output.go

```diff
@@ -102,11 +103,15 @@ func PrintOutputWithType(data interface{}, outputType string) error {
 		return dumpJSON(data, "")
 	}
 
+	if outputType == "yaml" {
```

Comment:
We should probably add `yaml` to the usage message in `AddJSONOutput` above as well. And while at it, it might make sense to rename that function to something like `AddOutputOption` as it's no longer about just JSON.